modules/gitops-enable-replicas-for-argo-cd-server.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * gitops-argo-cd-installation.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-enable-replicas-for-argo-cd-server_{context}"]
+= Enabling replicas for Argo CD server and Repo server
+
+To better distribute your workloads among pods, you can increase the number of Argo CD server repo-server replicas. If a horizontal autoscaler is enabled on the Argo CD-server, it overrides the number of replicas you set.
```

Comment:
- The title says: Enabling replicas for Argo CD server and Repo server
Given that, I wonder if the first sentence must read: '...you can increase the number of Argo CD server and repo-server replicas.'
- Repo/repo, pls make it consistent
- I think this should this be repository instead of Repo, please confirm with the devs. If yes, it would help to mention which repository?